### PR TITLE
Add auth0

### PR DESCRIPTION
**Pull request by JakubC-projects**
1. Adds token verification to all of the current endpoints
2. Removes POSTGRES_SSL_MODE env var as it is always in disabled mode (even in production)
3. Adds sample requests for easier testing

### CODE REVIEW

File: bcc-orgs/src/auth0/verify.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+func checkScopes(token jwt.Token, requiredScopes []string) bool {
+	scope, ok := token.Get("scope")
+	if !ok {
+		return false
+	}
+	tokenScopes := strings.Split(scope.(string), " ")
+	for _, requiredScope := range requiredScopes {
+		if !utilities.Contains(tokenScopes, requiredScope) {
```
Comment (StevenMalaihollo) [inline bcc-orgs/src/auth0/verify.go:58]: https://github.com/samber/lo

File: bcc-orgs/src/auth0/verify.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package auth0
```
Comment (StevenMalaihollo) [inline bcc-orgs/src/auth0/verify.go:1]: Maybe rename to auth

**Review by StevenMalaihollo — APPROVED** [review]

File: bcc-orgs/src/auth0/verify.go (hunk 1 above)
Comment (JakubC-projects) [reply bcc-orgs/src/auth0/verify.go]: done

File: bcc-orgs/src/auth0/verify.go (hunk 2 above)
Comment (JakubC-projects) [reply bcc-orgs/src/auth0/verify.go]: done

**Pull request merged**